TestProject/Packages/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "disableProjectUpdate": true,
+  "dependencies": {
+    "com.unity.build-report-inspector": "file:../../com.unity.build-report-inspector",
+    "com.unity.ide.rider": "2.0.2"
```

Comment:
Since you added rider, probably would be good to add visualstudio, vs code as well?

Reply:
I added this by accident, will remove 🙂